src/Build/Evaluation/Expander.cs

```diff
@@ -3116,6 +3132,14 @@ private static class RegularExpressions
             /// </summary>
             internal const string ItemTypeGroup = "ITEM_TYPE";
 
+#if NET7_0_OR_GREATER
+            [GeneratedRegex(@"((?<=" + ItemVectorWithTransformLHS + @")" + ItemMetadataSpecification + @"(?!" +
```

Comment:
Is it possible to move this string to some const and reuse it in these 2 places?
```
@"((?<=" + ItemVectorWithTransformLHS + @")" + ItemMetadataSpecification + @"(?!" +
                            ItemVectorWithTransformRHS + @")) | ((?<!" + ItemVectorWithTransformLHS + @")" +
                            ItemMetadataSpecification + @"(?=" + ItemVectorWithTransformRHS + @")) | ((?<!" +
                            ItemVectorWithTransformLHS + @")" + ItemMetadataSpecification + @"(?!" +
                            ItemVectorWithTransformRHS + @"))"
```